### PR TITLE
Add integration test with metrics from the prometheus receiver

### PR DESCRIPTION
**Pull request by dashpole**
It is generated by scraping the opentelemetry-go's prometheus example: https://github.com/open-telemetry/opentelemetry-go/blob/main/example/prometheus/main.go

I mostly wanted to make sure we have an example that uses a histogram, which we didn't previously have.

### CODE REVIEW

File: exporter/collector/integrationtest/testdata/fixtures/basic_prometheus_metrics_expect.json [hunk 1]
```diff
@@ -0,0 +1,1143 @@
...
+              "namespace": "",
+              "task_id": "localhost:2222"
+            }
+          },
+          "metricKind": "GAUGE",
+          "valueType": "DOUBLE",
+          "points": [
+            {
+              "interval": {
+                "endTime": "1970-01-01T00:00:00Z"
```
Comment (dashpole) [inline exporter/collector/integrationtest/testdata/fixtures/basic_prometheus_metrics_expect.json:98]: This seems odd.

Comment (dashpole) [reply exporter/collector/integrationtest/testdata/fixtures/basic_prometheus_metrics_expect.json:98]: ah, its just normalized in the test.

**Comment by dashpole** [comment]
this is ready for review

**Comment by dashpole** [comment]
Note to self: fix this error.
```
{"level":"error","ts":1649711917.6179736,"caller":"collector/metrics.go:292","msg":"Unable to send metric descriptor.","error":"rpc error: code = InvalidArgument desc = Field metricDescriptor.labels[3].key had an invalid value of \"service_name\": Duplicate label key encountered.","metric_descriptor":"name:\"ex_com_one\" type:\"workload.googleapis.com/ex_com_one\" labels:{key:\"service_name\"} labels:{key:\"service_instance_id\"} labels:{key:\"ex_com_lemons\"} labels:{key:\"service_name\"} labels:{key:\"telemetry_sdk_language\"} labels:{key:\"telemetry_sdk_name\"} labels:{key:\"telemetry_sdk_version\"} labels:{key:\"A\"} labels:{key:\"B\"} labels:{key:\"C\"} metric_kind:GAUGE value_type:DOUBLE display_name:\"ex_com_one\"","stacktrace":"github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector.(*MetricsExporter).exportMetricDescriptor\n\t/workspace/exporter/collector/metrics.go:292\ngithub.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector.(*MetricsExporter).exportMetricDescriptorRunner\n\t/workspace/exporter/collector/metrics.go:262"}
{"level":"error","ts":1649711917.6435902,"caller":"collector/metrics.go:292","msg":"Unable to send metric descriptor.","error":"rpc error: code = InvalidArgument desc = Field metricDescriptor.labels[3].key had an invalid value of \"service_name\": Duplicate label key encountered.","metric_descriptor":"name:\"ex_com_two\" type:\"workload.googleapis.com/ex_com_two\" labels:{key:\"service_name\"} labels:{key:\"service_instance_id\"} labels:{key:\"ex_com_lemons\"} labels:{key:\"service_name\"} labels:{key:\"telemetry_sdk_language\"} labels:{key:\"telemetry_sdk_name\"} labels:{key:\"telemetry_sdk_version\"} labels:{key:\"A\"} labels:{key:\"B\"} labels:{key:\"C\"} metric_kind:CUMULATIVE value_type:DISTRIBUTION display_name:\"ex_com_two\"","stacktrace":"github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector.(*MetricsExporter).exportMetricDescriptor\n\t/workspace/exporter/collector/metrics.go:292\ngithub.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector.(*MetricsExporter).exportMetricDescriptorRunner\n\t/workspace/exporter/collector/metrics.go:262"}
```

**Review by damemi — APPROVED** [review]

**Pull request merged**